### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by laeschjs**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,20 @@
...
+<artifactId>java-sdk</artifactId>
+<version>0.0.1</version>
+```
+
+## Example Usage
+
+As an example you can add a new user to a product to keep track of their quotas and billing.
+```java
+QuotaService quotaService = QuotaService.getReference(“API_KEY”);
+Product product = quotaService.getProductByName(“NAME”);  // or by ID
```
Comment (jackhallam) [inline README.md]: We switched ID to Id at some point

File: README.md [hunk 2]
```diff
@@ -1,3 +1,20 @@
...
-Java SDK
+# Quota/Billing Java SDK
+
+To start using our service visit [our quota-billing service](https://url_to_quota_billing_README.com) to generate an API key and upload your configuration file. Then add to your pom.xml or build.gradle file:
+
+```xml
+<groupId>edu.rosehulman.quota</groupId>
+<artifactId>java-sdk</artifactId>
+<version>0.0.1</version>
+```
```
Comment (jackhallam) [inline README.md]: Put the jitpack stuff here. Also put the outer <dependency> tags.

**Review by jackhallam — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (jackhallam) [reply README.md]: The outer tags are
```xml
<dependency>
</dependency>
```

File: README.md (hunk 1 above)
Comment (laeschjs) [reply README.md]: Do you mean NAME to Id?

Comment (jackhallam) [reply README.md]:
```java
Product product = quotaService.getProductByName(“NAME”);  // or by Id
```

**Review by jackhallam — APPROVED** [review]

**Pull request merged**